projects/packages/stats-admin/src/class-odyssey-config-data.php

```diff
@@ -79,9 +83,15 @@ public function get_data() {
 							'products'      => array(),
 							'plan'          => $empty_object, // we need this empty object, otherwise the front end would crash on insight page.
 							'options'       => array(
-								'wordads'    => ( new Modules() )->is_active( 'wordads' ),
-								'admin_url'  => admin_url(),
-								'gmt_offset' => $this->get_gmt_offset(),
+								'wordads'               => ( new Modules() )->is_active( 'wordads' ),
+								'admin_url'             => admin_url(),
+								'gmt_offset'            => $this->get_gmt_offset(),
+								'is_automated_transfer' => $host->is_woa_site(),
+								'is_wpcom_atomic'       => $host->is_woa_site(),
```

Comment:
What's your rationale for adding both of these options?

Reply:
In calypso the option has two selectors of two options. Added two just in case.

Reply:
For late comers: https://github.com/Automattic/wp-calypso/pull/76746#discussion_r1190436510